build/meson/meson.build

```diff
@@ -12,8 +12,11 @@ project('zstd',
   ['c', 'cpp'],
   license: ['BSD', 'GPLv2'],
   default_options : [
-    'c_std=gnu99',
-    'cpp_std=c++11',
+    # There shouldn't be any need to force a C or C++ standard convention
+    # but in case one would want that anyway,
+    # this can be done here.
+    # 'c_std=gnu99',
+    # 'cpp_std=c++11',
```

Comment:
Actually we do need to keep this line, since pzstd requires c++ >= 11.

Reply:
changed